wormhole-connect/src/utils/index.ts

```diff
@@ -373,6 +373,10 @@ export const isFrankensteinToken = (token: TokenConfig, chain: Chain) => {
   return (
     nativeChain !== chain &&
     !(['Ethereum', 'Sepolia'] as Chain[]).includes(nativeChain) &&
-    ['ETH', 'WETH', 'wstETH', 'USDT', 'USDC'].includes(symbol)
+    ['ETH', 'WETH', 'wstETH', 'USDT', 'USDC', 'USDC.e'].includes(symbol)
   );
 };
+
+export const isWrappedToken = (token: TokenConfig, chain: Chain) => {
+  return token.nativeChain !== chain;
```

Comment:
@kev1n-peters are all non-native tokens considered wrapped tokens?

Reply:
that's right - if a token is not native to the current chain, then it's wormhole-wrapped in Connect